posthog/api/plugin.py

```diff
@@ -107,17 +107,23 @@ def log_config_update_activity(
     log_enabled_change_activity(new_plugin_config=new_plugin_config, old_enabled=old_enabled, user=user)
 
 
-def _update_plugin_attachment(plugin_config: PluginConfig, key: str, file: Optional[UploadedFile]):
+def _update_plugin_attachment(plugin_config: PluginConfig, key: str, file: Optional[UploadedFile], user: Any):
```

Comment:
We should generally avoid the `Any` type. I think you did this because of type warnings on the callsite?

If so, you can do `user = cast(User, request.user)` in the callsite - this pattern is used elsewhere as well.

Reply:
I was following the existing pattern - other functions in this file take a user and say it's of type `Any`

I tried changing it but it seems to be a bit more intrusive, mypy thinks it should be `Union[User, AnonymousUser]` but it seems like we'd never have an anonymous user in this area. I'll fix this in another PR.